pages/agent/v3/cli_pipeline.md

```diff
@@ -48,7 +48,7 @@ If you want an environment variable to be evaluated at run-time (for example, us
     SERVER: "server-a"
 ```
 
-### Escaping the `$` character
+### Escaping the $ character
```

Comment:
Hmm, I can see why this is being changed, given how this title renders today:

![CleanShot 2024-04-16 at 14 52 19@2x](https://github.com/buildkite/docs/assets/251281/a0ec8834-268e-4c55-bd9f-82d8899e6a70)

The rendering of monospaced text inside titles feels like an issue to address regardless, though. Can we flag that for future work if we haven't already got it on our radar?

Reply:
Ah, those reasons make a lot of sense! In that case it's probably fine to just avoid this in future \o/